### PR TITLE
[15.0][3902][MIG] product_ext_sst: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3902](https://www.quartile.co/web#id=3902&cids=3&menu_id=505&action=1457&model=project.task&view_type=form)
Depends on: https://github.com/qrtl/rmm-custom/pull/98

### CODE REVIEW

File: product_ext_sst/views/product_template_views.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                <field name="standard_price" />
+                <field name="auction_start_price" />
+                <field name="auction_buyout_price" />
+                <field name="list_price" />
+                <field name="carrier_id" />
+                <field name="carrier_size_id" />
+                <field name="selectable_delivery_city_ids" invisible="1" />
+                <field name="delivery_state_id" />
+                <field name="delivery_city_id" />
+                <!-- <field name="yahoo_product_state_id" /> -->
```
Comment (kanda999) [inline product_ext_sst/views/product_template_views.xml]:
```suggestion
                <field name="yahoo_product_state_id" />
```
Why did you comment out?

File: product_ext_sst/views/product_template_views.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            </xpath>
+        </field>
+    </record>
+    <record id="sale_product_template_tree_view" model="ir.ui.view">
+        <field name="name">sale.product.template.product.tree</field>
+        <field name="model">product.template</field>
+        <field eval="20" name="priority" />
+        <field name="arch" type="xml">
+            <tree editable="top">
+                <field name="sequence" widget="handle" />
```
Comment (kanda999) [inline product_ext_sst/views/product_template_views.xml]:
```suggestion
                <field name="sequence" widget="handle" optional="show"/>
```
Can you add `optional="show"` to the fields in the tree view?

**Review by kanda999 — CHANGES_REQUESTED** [review]

File: product_ext_sst/views/product_template_views.xml (hunk 1 above)
Comment (AungKoKoLin1997) [reply product_ext_sst/views/product_template_views.xml]: I create this PR before we add yahoo_product_state_id in product_yahoo_auction_sst. I was not sure we use this field in rmm.

**Review by kanda999 — APPROVED** [review]
Code and functional review.

**Pull request merged**